### ISSUE TITLE
captureUnhandledException lacks scope information

### ISSUE THREAD

**Issue by scanningdeer** [getsentry/sentry-laravel#615]
### Environment
Debian
PHP 8.1.6 
Laravel-Sentry 3.1.0
Sentry 3.3
### Steps to Reproduce

1.  Use guide from https://docs.sentry.io/platforms/php/guides/laravel/
and add following to app/Exceptions/Handler.php
```
public function register()
{
    $this->reportable(function (Throwable $e) {
        Integration::captureUnhandledException($e);
    });
}
```

2. Inside a local controller throw an exception directly.

3. Go to sentry and observe that the event lacks expected breadcrumbs and surrounding context for PHP code.  

### Expected Result

Expected breadcrumbs and context lines for stack trace like we get when using 
```
app('sentry')->captureException($e);
``` 

### Actual Result

No breadcrumbs / context lines.

### Observed Cause
It seems that app('sentry')->captureException($e); ends up calling Hub->captureException() that manages to inject $scope into the Client->captureException().  

I believe letting the captureUnhandledException use the hub's captureException instead of the client's captureException will fix this.

**Comment by scanningdeer** [comment]
This looks to be fixed yesterday in 3.1.1

**Comment by stayallive** [comment]
Exactly, but great analysis, spot on! 👌
